Scripts/Runtime/Entities/TaskDriver/TaskSet/Job/Wrapper/GenericExclusiveWriteDataAccessWrapper.cs

```diff
@@ -0,0 +1,32 @@
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities.TaskDriver
+{
+    internal class GenericExclusiveWriteDataAccessWrapper<TData> : AbstractAccessWrapper
+        where TData : struct
+    {
+        private readonly IExclusiveWriteAccessControlledValue<TData> m_AccessControlledData;
+        private TData m_Data;
+
+        public TData Data
+        {
+            get => m_Data;
+        }
+
+        public GenericExclusiveWriteDataAccessWrapper(IExclusiveWriteAccessControlledValue<TData> data, AccessType accessType, AbstractJobConfig.Usage usage) : base(accessType, usage)
```

Comment:
```suggestion
        public GenericExclusiveWriteDataAccessWrapper(IExclusiveWriteAccessControlledValue<TData> data, AbstractJobConfig.Usage usage) : base(AccessType.ExclusiveWrite, usage)
```

Reply:
Might want to check the SharedWrite and Read versions of this type of wrapper. Less error prone.

Reply:
Fixed